docs/guide/env-and-mode.md

```diff
@@ -46,10 +46,18 @@ DB_PASSWORD=foobar
 Only `VITE_SOME_KEY` will be exposed as `import.meta.env.VITE_SOME_KEY` to your client source code, but `DB_PASSWORD` will not.
 
 ```js
-console.log(import.meta.env.VITE_SOME_KEY) // 123
+console.log(import.meta.env.VITE_SOME_KEY) // "123"
 console.log(import.meta.env.DB_PASSWORD) // undefined
 ```
 
+:::tip Env parsing
+
+As seen in example above `VITE_SOME_KEY` is a number.  
+But when parsed, Env properties are strings even if they are numbers or booleans in the Env file.  
+User has to convert them to desired types in the code.
+A recommandation is to wrap all the Env properties in quotes to make that explicit.
+:::
```

Comment:
```suggestion
:::tip Env parsing

As shown above, `VITE_SOME_KEY` is a number but returns a string when parsed. The same would also happen for boolean env variables. Make sure to convert to the desired type when using it in your code.
:::
```

I think we can shorten this a bit, and keep it one line for consistency. I also removed the recommendation for now since I think reminding about about converting should be enough.